@import "CV.css"

Jiawei Gu
============
LinkedIn:   https://www.linkedin.com/in/jiawei-gu/ | GitHub: https://github.com/jiawku
Cell: (801)-637-­4021 | Email:     jiawku17@gmail.com | Salt Lake City, UT

Summary
------
I am bioinformatician with fours years of experience in academic and industry. I obtained a Master in biology and a Master in computer science, so that I can combine biology knowledge with programming, data engineering ,and analysis.

Work Experience
----------
**Bioinformatics Analyst, Navcian Genomics, Inc. Salt Lake City, 08/2018-Present**

I work corporately with Lab Operation Team, Variant Scientist Team, DevOps Team ,and R&D team in Navcian Genomics. My job mostly consisted of three parts:
 * ***Production & Automation***
    * Daily clinical control sequencing quality assurance.
    * Designed and implemented the **Tumor Mutation Burden(TMB)** calculation algorithm within the bioinformatics team.
    * Built an **automated pipeline** for clinical control assurance in **python**. It connected multiple **cloud service platforms**(Philips IntelliSpace Genomics Medicine, SalesForce, SeraCare IQ, Slack, and Amazon AWS) via **API**.
    * Created an **Illumina BaseSpace Native APP** for sequencing quality control using **Docker**.  
 * ***Data Engineering***
    * Created tools to query and clean data from **cloud service platforms** (Philips IntelliSpace Genomics Medicine, SalesForce, Clarity LIMS) via **API** for ETL and R&D.
    * Designed a Bioinformatics **Variant Database** to store clinical mutations.
 * ***Data Analysis***
    * Implemented and run validation workflows for Illumina TruSight™ Tumor 170 Sequencing Assay.
    * Designed statistical tests for lab operation quality metrics.
    * Help with other ad-hoc data analysis requests.


**Research Assistant, Biology Department, UTD 	01/2014-12/2016**

I focused on the research of Next-Generation Sequencing, especially on ChIP-seq, RNA-seq, and ChIA-PET Seq result analysis, and using machine learning tools to analysze and predict from data.

Publication: [Shi, X., et al. "SMARCA4/Brg1 coordinates genetic and epigenetic networks underlying Shh-type medulloblastoma development." Oncogene (2016)](https://www.nature.com/articles/onc2016108).

Skills
--------
Programming languages
:   Python, Java, JavaScript, R, SAS, SQL

Machine Learning & Statistical model
:   Support Vector Machine(SVM), Random Forest, Deep Learning, Hidden Markov Model(HMM), Expectation Maximization(EM)

Frameworks & Tools
:   BigData:   Apache Spark, Apache Hadoop
    Web:   MEAN stack(MongoDB, Express JS, Angular JS, Node.js), Java Spring
    Deep Learning:   TensorFlow, Keras
    Containerization: Docker, Kubernetes

Cloud Service Platforms
: Amazon AWS, SalesForce

Education
---------

01/2017-05/2018
: **MS, Computer Science**; University of Texas at Dallas(UTD); **3.91/4**

08/2013-12/2016
: **MS, Bioinformatics**; University of Texas at Dallas(UTD); **3.73/4**

Certification
--------

*   [SAS Certified Base & Advanced Programmer for SAS 9](https://www.youracclaim.com/badges/29649285-43db-4ca4-89c6-5e325a059829/public_url)
*   [edX Verified Certificate for Big Data Analysis with Apache Spark](https://courses.edx.org/certificates/user/10309856/course/course-v1:BerkeleyX+CS110x+2T2016)
*   [Coursera Deep Learning Specialization](https://www.coursera.org/account/accomplishments/specialization/5VL96NJ4AGXJ)
-------------------    -------------------
